data_sources/sysmon_eventid_22.yml

```diff
@@ -95,6 +95,14 @@ fields:
 - timestartpos
 - user_id
 - vendor_product
+output_fields:
+- answer 
```

Comment:
lets add

process_exec
process_name
process_path

Reply:
For Sysmon Event ID 22 it is not possible to add these to the output as they are not part of the DNS CIM data model